src/context/index.tsx

```diff
@@ -18,25 +18,33 @@ type FoundryColorsType = Record<keyof typeof colorsEnum, string>;
 export type FoundryContextType = {
   globalStyles: string;
   colors: FoundryColorsType;
+  styleConstants: object;
```

Comment:
I think the type here should be more like 
```suggestion
  styleConstants?: {[key in string]: number | string};
```

I probably got that wrong but you get the idea. TS or eslint should have complained about the use of `object` here